pkg/operator/ceph/cluster/nodedaemon/exporter.go

```diff
@@ -278,3 +278,10 @@ func applyPrometheusAnnotations(cephCluster cephv1.CephCluster, objectMeta *meta
 		t.ApplyToObjectMeta(objectMeta)
 	}
 }
+
+func generateExporterEnvVar() corev1.EnvVar {
+	val := fmt.Sprintf("-m $(ROOK_CEPH_MON_HOST) -k %s", keyring.VolumeMount().ExporterKeyringFilePath())
```

Comment:
```suggestion
	val := fmt.Sprintf("-m $(ROOK_CEPH_MON_HOST) -n client.ceph-exporter -k %s", keyring.VolumeMount().ExporterKeyringFilePath())
```

Reply:
Thanks @travisn for testing it! I've updated the PR with changes you mentioned.